lib/smart_todo/events/date.rb

```diff
@@ -3,14 +3,14 @@
 require "time"
 
 module SmartTodo
-  module Events
+  class Events
     # An event that check if the passed date is passed
     class Date
       class << self
         # @param on_date [String] a string parsable by Time.parse
         # @return [String, false]
-        def met?(on_date)
-          if Time.now >= Time.parse(on_date)
+        def met?(on_date, now = Time.now)
```

Comment:
Do we even use this method without the second argument?

Reply:
We don't, I just wasn't sure how much you wanted to break the existing APIs. Happy to make this required if you're prefer.